src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMassesGenerator.cpp

```diff
@@ -435,48 +481,48 @@ namespace OpenMS
     // sort mass patterns
     // (from small mass shifts to larger ones, i.e. few miscleavages = simple explanation first)
     std::sort(delta_masses_list_.begin(), delta_masses_list_.end());
-
+    
   }
   
   void MultiplexDeltaMassesGenerator::printSamplesLabelsList() const
   {
-    LOG_INFO << "\n";
+    std::cout << "\n";
     for (unsigned i = 0; i < samples_labels_.size(); ++i)
     {
-      LOG_INFO << "sample " << (i + 1) << ":    ";
+      std::cout << "sample " << (i + 1) << ":    ";
       for (unsigned j = 0; j < samples_labels_[i].size(); ++j)
       {
-        LOG_INFO << samples_labels_[i][j] << "    ";
+        std::cout << samples_labels_[i][j] << "    ";
       }
-      LOG_INFO << "\n";
+      std::cout << "\n";
     }
   }
   
   void MultiplexDeltaMassesGenerator::printDeltaMassesList() const
```

Comment:
you could pass the output stream as parameter so the user could choose if it is written to std::out, a stringstream, a file, or log_info/debug ...

Reply:
no that's not what I mean.
you add a parameter to your method: print...(ostream& theStream)
then the user can decide which stream get's the output.
you can search for "passing stream as parameter c++" to see some examples

Reply:
Done. Much nicer. Thanks.